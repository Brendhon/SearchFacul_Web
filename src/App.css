@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;600;700&display=swap');
/* Declaração de Variáveis */

:root {
    --c-main: #00A5D8;
    --b-card: #4b86b4;

    --c-dark: #2E333E;
    --c-red: #c82333da;
}

/* Definindo alguns padrões de CSS para todos CSS utilizados */

* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

/* Criando alguns padrões de CSS */

body {
    font-family: Nunito, sans-serif;
    background-color: #d9e7e9;
}

.form-field {
    font-family: Nunito, sans-serif;
    width: 100%;
    height: 50px;
    font-size: 1rem;
    color: var(--c-dark);
    border: 0;
    border-radius: 8px;
    padding: 0 24px;
    outline: none;
}

.form-error  {
    font-size: 0.8rem;
    color: red;
    margin-left: 5px;
}

.form-textarea {
    font-family: Nunito, sans-serif;
    width: 100%;
    min-height: 140px;
    resize: vertical;
    color: var(--c-dark);
    font-size: 1rem;
    border: 0;
    border-radius: 8px;
    padding: 16px 24px;
    line-height: 24px;
    outline: none;
}

.form-field:focus, .form-textarea:focus {
    border: 1px solid var(--c-main);
}

.button {
    display: flex;
    justify-content: center;
    align-items: center;
    background: var(--c-main);
    border: 0;
    border-radius: 8px;
    text-decoration: none;
    color: #fff;
    font-weight: 600;
    font-size: 18px;
    padding: 0px 10px 0px 10px;
    height: 50px;
    outline: none;
    cursor: pointer;
}

.button:active {
    opacity: 0.8;
}

.box {
    background: #f0f0f6;
    box-shadow: 0 0 50px rgba(0, 0, 0, 0.3);
    border-radius: 8px;
    margin: 0 3%;
}

.container {
    display: flex;
    flex-direction: column;
    height: 100vh;
}

.content-column {
    display: flex;
    flex: 0.8;
    flex-direction: column;
}

.content-row {
    display: flex;
    flex: 0.8;
}

.a-link {
    font-size: 0.9rem;
    background: transparent;
    border: 0;
    color: var(--c-main);
    outline: none;
    cursor: pointer;
    text-decoration: none;
}

.a-link:hover {
    text-decoration: underline;
}

.button-outline {
    background: transparent;
    font-size: 18px;
    padding: 0px 10px 0px 10px;
    height: 50px;
    outline: none;
    font-weight: normal;

    border: 1px solid var(--c-main);
    color: var(--c-main);
    border-radius: 8px;

    cursor: pointer;
}

.button-outline:hover {
    background: var(--c-main);
    color: white;
}

.fadeIn {
    -webkit-animation: fadein 0.5s; /* Safari, Chrome and Opera > 12.1 */
    -moz-animation: fadein 0.5s; /* Firefox < 16 */
    -ms-animation: fadein 0.5s; /* Internet Explorer */
    -o-animation: fadein 0.5s; /* Opera < 12.1 */
    animation: fadein 0.5s;
}

@keyframes fadein {
    from { opacity: 0; }
    to   { opacity: 1; }
}

/* Firefox < 16 */
@-moz-keyframes fadein {
    from { opacity: 0; }
    to   { opacity: 1; }
}

/* Safari, Chrome and Opera > 12.1 */
@-webkit-keyframes fadein {
    from { opacity: 0; }
    to   { opacity: 1; }
}

/* Internet Explorer */
@-ms-keyframes fadein {
    from { opacity: 0; }
    to   { opacity: 1; }
}

/* Opera < 12.1 */
@-o-keyframes fadein {
    from { opacity: 0; }
    to   { opacity: 1; }
}