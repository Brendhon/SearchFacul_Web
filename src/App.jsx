import React from 'react'
import Logon from './pages/Logon'
import './App.css'

function App() {
  return (
    <div className="main">
      <Logon />
    </div>
  )
}

export default App;
