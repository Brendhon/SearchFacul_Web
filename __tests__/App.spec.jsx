import React from 'react'
import { shallow } from 'enzyme'

import App from '../src/App'

describe("App", () => {

    it("Should be able to render", async () => {
        const wrapper = shallow(
            <App/>
        )
        expect(wrapper).toMatchSnapshot()
    })
})