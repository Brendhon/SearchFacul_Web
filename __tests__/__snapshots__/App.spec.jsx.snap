// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should be able to render 1`] = `ShallowWrapper {}`;
